### ISSUE TITLE
 Fix broken link on p5.js website page for "How to add Friendly Error Messages"

### ISSUE THREAD

**Issue by iamniky** [processing/p5.js-website#512]
### Most appropriate sections of the p5.js website?

Home

### What is your operating system?

Windows

### Web browser and version

Firefox v130.0, though would affect all.

### Actual Behavior

When users visit the URL https://p5js.org/contribute/how-to-add-friendly-error-messages/ and click on the hyperlink that is labeled "Unit Testing and Test Driven Development", they are met with an access denied page:

![Screenshot 2024-09-06 010525](https://github.com/user-attachments/assets/fe898c24-6e12-4db2-87f4-a1b1ab944c9f)

![Screenshot 2024-09-06 010557](https://github.com/user-attachments/assets/e1a4eff1-6421-419d-b1d3-9ec5f1e78ff9)

### Expected Behavior

We expect that when the users click the hyperlink labeled "Unit Testing and Test Driven Development", they are taken to a page with information regarding unit testing and test driven development.

### Steps:

1. Go to https://p5js.org/contribute/how-to-add-friendly-error-messages/
2. Click on respective links (listed in table below).
3. Get a 404 error.

### Solution:

Replace listed broken links to the correct links. This can be achieved by changing the access permissions on the document.

Source link text: | Current link: | Suggested link:
-- | -- | --
Unit Testing and Test Driven Development | https://docs.google.com/document/d/1iw60L1abRJZDn1zUViO9A0j6Fco4yFoQ9LXIlEEroN8/edit?usp=sharing | ???

### Would you like to work on the issue?

Yes, I'd like to work on the issue.

**Comment by shourysingh07** [comment]
I'd like to work on the issue.

**Comment by Qianqianye** [comment]
Thanks @iamniky for catching this bug and @shourysingh07 for your interest in working on this.

I'm bringing @limzykenneth, who worked on the "How to add Friendly Error Messages" doc before, to this discussion. I believe the 'Unit Testing and Test Driven Development by Andy Timmons' text might supposed to go to this archived tutorial on the legacy p5.js website: https://archive.p5js.org/learn/tdd.html. Shall we keep the link?

**Comment by limzykenneth** [comment]
The document "[Unit Testing and Test Driven Development](https://docs.google.com/document/d/1iw60L1abRJZDn1zUViO9A0j6Fco4yFoQ9LXIlEEroN8/edit?usp=sharing) by Andy Timmons" that is linked to here is the same content as the second link "[Contributors Doc: Unit Testing](https://p5js.org/contribute/unit_testing/)", both of which I wrote during the content update (the Google doc one is where I wrote it and the contributor docs one is after we migrated it).

The correct link as @Qianqianye mentioned should be pointing to the archived tutorial. We can correct the link and keep it around for now.
